Makefile

```diff
@@ -68,6 +68,7 @@ lessons:
 site :
 	bundle exec jekyll build
 	find _data -name '*.json' -exec cp {} _site/ \;
+	find images -iregex ".*.\(svg\|html\|png\)" -exec cp {} _site/ \;
```

Comment:
Would JPG/JPEGs be included here?

Reply:
We don't have any in our current feeds but certainly could, so I have added them here and to gitignore.